schema.sql

```diff
@@ -9,3 +9,21 @@ CREATE TABLE animals (
    weight_kg DECIMAL
 );
 ALTER TABLE animals ADD COLUMN species VARCHAR(250);
+
+CREATE TABLE owners(
+   id SERIAL PRIMARY KEY,
```

Comment:
- [ ] <b>OPTIONAL:</b>   It would be nice to add `NOT NULL` constraint on ID just to make sure that it can't accept null values.